flexible_event_config.md

```diff
@@ -0,0 +1,303 @@
+
+Note that these matching criteria take place _after_ attributing a trigger to a particular source i.e. after step 5 [here](https://wicg.github.io/attribution-reporting-api/#trigger-attribution). If `trigger_specs` are specified for a source and no matching spec is found after attribution, the trigger will be ignored. 
+
+Triggers will also support adding an optional “value” parameter in the dictionaries within `event_trigger_data`.
+
+```jsonc
+{
+  "event_trigger_data": [
+    {
+      "trigger_data": "2",
+      "value": 100,  // Defaults to 1
+      "filters": ...
+    },
+    ...
+  ]
+}
+```
+
+Every trigger registration will match with at most one trigger spec and update its associated summary value. At a high level, at trigger time we will:
+
+* Apply global attribution filters
+* For every trigger spec:
+  * Evaluate the `event_trigger_data` on the spec to find a match, using the spec’s `event_reporting_window`
+* The first matched spec is chosen for attribution, and we increment its summary value by `value`.
+
+When the `event_report_window` for a spec completes, we will map it's summary value to a bucket, and send an event-level report for every increment in the summary bucket caused by attributed trigger values. Reports will come with one extra field `trigger_summary_bucket`.
+
+```jsonc
+{
+  ...
+  "trigger_summary_bucket": [<bucket start>, <bucket end>],
+}
+```
+
+
+## Privacy considerations
+
+The goal of this proposal is to remain largely privacy neutral with respect to the existing event-level reports. We will publish an algorithm which computes the number of output states for a given source registration. From this we will be able to:
+
+* Compute a randomized response algorithm across the entire output space
+* Set the noise level to satisfy a certain epsilon level level via a randomized response mechanism
+* Verify that the privacy parameters (like information gain) are within a given threshold, and fail registration if they are not
+
+With these pieces we can ensure that this extension does not regress our privacy measures. Additionally, it allows callers to fine-tune the noise added to the API e.g. by specifying different kinds of output domains. For example, `navigation` sources that only need 1 bit of trigger data and 1 reporting window can use the same noise level as `event` sources.
+
+Beyond setting noise levels, we will have some parameter limits to avoid large computation costs and avoid configurations with too many output states (where noise will increase considerably). Here is an example set of restrictions (feedback always welcome):
+
+* Maximum of 20 total reports, globally and per `trigger_data`
+* Maximum of 5 possible reporting windows per `trigger_data`
+* Maximum of 32 for trigger data cardinality
```

Comment:
'Maximum of 32 for trigger data cardinality' to 'Maximum of 32 trigger data cardinality'

Reply:
done